vendor/README.md

```diff
@@ -390,6 +390,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 - **Squirrel:** [textmate/c.tmbundle](https://github.com/textmate/c.tmbundle)
 - **Stan:** [jrnold/atom-language-stan](https://github.com/jrnold/atom-language-stan)
 - **Standard ML:** [textmate/standard-ml.tmbundle](https://github.com/textmate/standard-ml.tmbundle)
+- **Starlark:** [MagicStack/MagicPython](https://github.com/MagicStack/MagicPython)
```

Comment:
Hmm, typo?

Reply:
@jin Not a typo. The list is auto-generated from whatever grammars are assigned to a language, which in this case happens to be Python.